Peer.GitHub/GraphQL/PullRequestSearch/CheckRunStatusState.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace Peer.GitHub.GraphQL.PullRequestSearch
+{
+    public enum CheckRunStatusState
+    {
+        Queued,
+        In_Progress,
+        Completed,
+        Waiting,
+        Pending,
+        Requested
+    }
+#nullable enable
```

Comment:
? Aren't these usually following a `#nullable disable` in the same file?

Reply:
same

Reply:
fixed